feature/schedule/src/main/java/com/yapp/feature/schedule/ScheduleViewModel.kt

```diff
@@ -60,6 +69,24 @@ class ScheduleViewModel @Inject constructor(
             }
     }
 
+    private fun loadUpcomingSessionInfo(
+        reduce: (ScheduleState.() -> ScheduleState) -> Unit
+    ) = viewModelScope.launch {
+        scheduleRepository.getUpcomingSessions()
+            .collectLatest { upcomingSessionInfo ->
+                reduce { copy(upcomingSessionInfo = upcomingSessionInfo) }
+            }
+    }
```

Comment:
```suggestion
    ) = 
        scheduleRepository.getUpcomingSessions()
        .onEach {}
        .catch {}
        .launchIn(viewModelScope)
```

이렇게 하면 collect에서 발생하는 예외도 처리할 수 있어요